frontend/src/exporter/index.html

```diff
@@ -2,11 +2,22 @@
 <html>
     <head>
         <meta name="robots" content="noindex" />
+        {{ exported_data|json_script:"posthog-exported-data" }}
```

Comment:
this is ultimately the fix for the reported issue. `json_script` is a better thing for us to be using than `|safe` for getting JSON data into the page